infrastructure/main.tf

```diff
@@ -62,3 +72,11 @@ module "ccd-case-print-service" {
     IDAM_SERVICE_NAME = "${var.idam_service_name}"
   }
 }
+
+// Copy into Azure Key Vault
+
+resource "azurerm_key_vault_secret" "idam_service_key" {
+  name = "idam-service-key"
```

Comment:
Conflicts with `ccd-api-gateway/idam-service-key` and `ccd-admin/idam-service-key`